lang/java/tools/src/test/java/org/apache/avro/tool/TestConcatTool.java

```diff
@@ -136,6 +137,38 @@ public void testConcat() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
Ignore should have a short reason included and optimally a JIRA (in this case "see note in comment" probably suffices)